mcp/pacemaker.c

```diff
@@ -1008,6 +1014,12 @@ main(int argc, char **argv)
         crm_exit(DAEMON_RESPAWN_STOP);
     }
 
+    if(start_standby) {
+        setenv("PCMK_node_start_state", "standby", 1);
+    } else {
+        setenv("PCMK_node_start_state", "default", 1);
+    }
```

Comment:
We should use set_daemon_option() instead of setenv() here. It's mostly equivalent, but it gives us more flexibility to change the options implementation in the future, if the need arises.